ci/launch_cluster.sh

```diff
@@ -38,9 +38,12 @@ template_parameters:
     AdminLocation: 0.0.0.0/0
     PublicSlaveInstanceCount: 1
     SlaveInstanceCount: 5
+    LicenseKey: $DCOS_LICENSE
```

Comment:
What is this is empty, e.g. with open?

Reply:
empty string should be ok, that is actually the default, but the open template might not contain this param - I hope it will just ignore it